### ISSUE TITLE
Unable to add NuGet package - fails to get package details.

### ISSUE THREAD

**Issue by mrward** [mrward/monodevelop-paket-addin#5]
If I try to install a NuGet package using this Paket addin I get an error similar to the following in the Paket Console.

```
Adding Newtonsoft.Json to ~/Projects/Tests/TestPaket/paket.dependencies
Resolving packages:
 - Newtonsoft.Json 9.0.1
Couldn't get package details for package Newtonsoft.Json on any of ["https://nuget.org/api/v2"].

Could not add Newtonsoft.Json.
```

**Comment by leolorenzoluis** [comment]
Experienced this too. Any reason why?

**Comment by mrward** [comment]
The addin is using an older version of Paket and I believe somethiing changed on the NuGet server side which broke Paket. Newer versions of Paket seem to work but there is a [bug](https://github.com/fsprojects/Paket/issues/1950) which is currently causing the addin problems when you try to remove a NuGet package.

**Comment by mrward** [comment]
Updating to the latest Paket fixes this. Released a new 0.2 version of the addin for MonoDevelop 6.0.
